### DISCUSSION TITLE
Can OpenTabletDriver make one pen button do scrolling like xsetwacom?

### DISCUSSION THREAD

**Discussion by zoliky** [OpenTabletDriver/OpenTabletDriver#3102] [General]
I have a Wacom CTL-4100. Can I use OpenTabletDriver to scroll websites by holding a pen button and moving up or down? And can I change how fast it scrolls? I had these functions with xsetwacom, but with my switch to Wayland, xsetwacom doesn't work anymore.

**Comment by jamesbt365** [comment]
we do not have independent scroll bindings at the moment, see #1381.

However, most desktop environments do support scroll by middle click (then moving your mouse to a different part of your screen to do what direction and speed) so you can use that for now in most cases.
